packages/instantsearch.js/src/connectors/trending-items/__tests__/connectTrendingItems-test.ts

```diff
@@ -129,8 +129,9 @@ describe('connectTrendingItems', () => {
         new RecommendParameters().addTrendingItems({
           // @ts-expect-error
           $$id: widget.$$id,
-          facetName: undefined,
-          facetValue: undefined,
+          // v5 expects only string values for facetName and facetValue
```

Comment:
Isn't it wrong though? IIRC you could choose not to provide `facetName` and `facetValue` .

Reply:
you're right, I forgot I already noted that and @shortcuts fixed it before the 5.0.0 release :)